code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -27,6 +27,7 @@
 	var/stop_automated_movement = 0 //Use this to temporarely stop random movement or to if you write special movement code for animals.
 	var/wander = 1	// Does the mob wander around when idle?
 	var/stop_automated_movement_when_pulled = 1 //When set to 1 this stops the animal from moving when someone is pulling it.
+	var/list/colourmatrix = list()
```

Comment:
This may be petty, but please use american spellings.

Reply:
Not petty at all; I had a nasty bug rebasing this yesterday that would have been entirely avoidable with consistent spelling conventions.

Reply:
Actually, looking at it now 'colour' pops up all over the place, possibly even more so than 'color'. Not sure what convention to follow now.

Reply:
Status update: SOMEONE. KILL. ME. And then a whole bunch of upstream contributors. Pretty please?

DM builtins consistently use `color`. Upstream code extensively uses `colour`. I am tempted to adopt `cololour` as the compromise Official Urist Convention.